prow/istio-presubmit.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
```

Comment:
a short description of what this does and how maybe would be a useful comment/header ?